subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -12,6 +12,26 @@ config BT_MESH_NRF_MODELS
 	  Common mesh model support modules, required by all Nordic BT mesh
 	  models.
 
+menu "Configuration of the waiting time for acknowledged messages"
+	depends on BT_MESH_NRF_MODELS
+	
+comment "The Bluetooth Mesh Profile recommends setting this timeout to a minimum of 30 seconds."
```

Comment:
"this timeout", What is really "this" here? Should the comment be moved to help in base.

Reply:
`this timeout` = `base timeout`  + (`per-hop timeout` * `number of hops`)
I do not think that adding this comment into the base timeout help message is correct.
Kconfig has quite limited syntax abilities. I do not know how to create multiline comment for the whole `menu` context.

Reply:
Simplified phrase a bit